spec/classes/rabbitmq_spec.rb

```diff
@@ -190,7 +190,8 @@
             is_expected.to contain_archive('rabbitmqadmin').with_source('http://1.1.1.1:15672/cli/rabbitmqadmin')
           end
 
-          it { is_expected.to contain_package('python') } if %w[RedHat Debian SUSE Archlinux].include?(os_facts[:os]['family'])
+          it { is_expected.to contain_package('python') } if %w[RedHat SUSE Archlinux].include?(os_facts[:os]['family'])
+          it { is_expected.to contain_package('python3') } if %w[Debian].include?(os_facts[:os]['family'])
```

Comment:
I played with single quoting this, but decided to keep the existing style, even for ones where we only have 1 right now.